client/app/js/controllers/main.js

```diff
@@ -576,15 +551,7 @@ controller('IntroCtrl', ['$scope', '$rootScope', '$uibModalInstance', function (
     $uibModalInstance.close();
   };
 
-  $scope.data = {
-    'language': $scope.language
-  };
-
-  $scope.$watch("data.language", function (newVal, oldVal) {
-    if (newVal && newVal !== oldVal) {
-      $rootScope.language = $scope.data.language;
-    }
-  });
+  $scope.GLTranslate = GLTranslate;
```

Comment:
probably this is the only line needed that would make every child scope inherit access to the object.